service/storage/operations_internal_test.go

```diff
@@ -927,7 +931,7 @@ func TestSaveAndRetrieve_Collection(t *testing.T) {
 			codec: codec,
 		}
 
-		err := db.Update(l.SaveCollection(mocks.GenericCollection(0)))
+		err := db.Update(l.SaveCollection(coll))
```

Comment:
Not sure this is super relevant, but `collection` is generally not shortened in the Flow codebase, while `collID` is.

Reply:
Even in `for` loops?

If you want I can go across the codebase and replace all occurrences of `coll` with `collection`. No problem on my side.

Reply:
It's not that relevant, but it's more consistent. `coll` is just too specific to the Flow domain to be obvious. I think `tx` and such are pretty clear.